build

```diff
@@ -85,3 +86,23 @@ ignore_list = [
 # Copy python repository tree inside dist folder
 copytree(PY_DIRECTORY, QUERY_MODULES_DIRECTORY, ignore_list)
 os.environ["PYTHONPATH"] = PY_DIRECTORY
+
+#################################################
+#              Load Rust modules                #
+#################################################
+
+# List all build modules in a list explicitly.
+# NOTE: Cargo prefixes `.so` files with the `lib` prefix (not possible to
+#       change).
+# NOTE: There are more files with the suffix `libmodule_name.so` under the Rust
+#       build directory.
+RS_MODULES = [
```

Comment:
If non relevant libnames.so are under the Rust build directory, it would be more suitable to put them in ignore, as in script above. Also, if each time library starts with "lib", there could be a rule to rename it, rather than giving the name explicitely.